strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/storage/checksum/ChecksumCacheManagerTest.java

```diff
@@ -1,150 +0,0 @@
-package org.carlspring.strongbox.storage.checksum;
```

Comment:
we shouldn't drop this

Reply:
This test is `@Ignore`ed in master.

Reply:
maybe, but we should fix it then instead of dropping